src/sidebar/store/modules/links.js

```diff
@@ -22,6 +22,13 @@ function updateLinksAction(newLinks) {
   return { type: 'UPDATE_LINKS', newLinks: newLinks };
 }
 
+/**
+ * @typedef LinksStore
+ *
+ * // Actions
+ * @prop {typeof updateLinks} updateLinksAction
```

Comment:
```suggestion
 * @prop {typeof updateLinks} updateLinks
```

The action creator is renamed in the export as `updateLinks`. The ideal solution here would be to change this store module's structure to match the others, then the action creator function can be named `updateLinks` and the confusion avoided.

Reply:
I think its a good time to do this in fact